index.html

```diff
@@ -71,15 +71,109 @@ <h2 class="small-name">Software Developer</h2>
 </li>
 </ul>
 </section>
+<section class="main-icons">
+    <div class="arrow-left">
+        <img src="/img/left.png" alt="">
+    </div>
+    <ul class=social-list>
+        <li class="social-it"><a href="https://github.com/iammouaz" class="fa fa-github fa-lg"></a></li>
+        <li class="social-it"><a href="https://www.linkedin.com/in/mohammad-mouaz-molki-1368981bb/" class="fa fa-linkedin fa-lg"></a></li>
+        <li class="social-it"><a href="#" class="fa fa-twitter fa-lg"></a></li>
+        <li class="social-it"><a href="#" class="fa fa-medium fa-lg"></a></li>
+         </ul>
+    
+     <div class="arrow-right"><img src="/img/right.png" alt=""></div>
+    </section>
+    <section class="my-works">
+        <p class="lets">My Recent Works</p>
```

Comment:
- [ ] The title (e.g., "My Recent Works") uses a header tag hx, use h1 for this not p